packages/docs/cookbook/create-store-dynamically.md

```diff
@@ -0,0 +1,51 @@
+# Create Store Dynamically
+
+Store can be created dynamically using factory pattern. Which will greatly help reduce boilerplate and structure your application.
+
+## Example
+
+An usecase of multiple tables that require pagination and filter. And all need to have separate stores to keep track of the result and pagings.
+
+We can have a creator function like this
+
+```js
```

Comment:
In this example, there should be `ts` if you want to use type annotations. Nothing critical, but the formatting would be a little bit better